lib/googleutil.js

```diff
@@ -7,35 +7,46 @@ let API_KEY = process.env['GOOGLE_PLACES_API_KEY'];
 
 
 //additionally to our locations we search google
-fns.searchNearbyPlaces = (request, reply) => {
-    let lat = request.params.lat;
-    let long = request.params.long;
-    let radius = request.params.maxDistance;
-
-    let googleURL = 'https://maps.googleapis.com/maps/api/place/nearbysearch/json?&location=' + lat + ',' + long + '&radius=' + radius + '&key=' + API_KEY;
-
-    Wreck.get(googleURL, {json: true}, (err, res, payload) => {
-        if (err) {
-            console.log('nearby error');
-            reply([]);
-        }
-        let googleArr = [];
-
-        payload.results.forEach(location => {
-            googleArr.push({
-                '_id': location.place_id,
-                'title': location.name,
-                'geotag': {
-                    'type': 'Point',
-                    'coordinates': [
-                        location.geometry.location.lat,
-                        location.geometry.location.lng
-                    ]
+fns.searchNearbyPlaces = (request) => {
+    return new Promise((resolve, reject) => {
+
+        let lat = request.query.lat;
+        let long = request.query.long;
+
+        let radius = 100;
+
+        let googleURL = 'https://maps.googleapis.com/maps/api/place/nearbysearch/json?&location=' + lat + ',' + long + '&radius=' + radius + '&key=' + API_KEY;
+
+        Wreck.get(googleURL, {json: true}, (err, res, payload) => {
+            if (err) {
+                reject(err);
+                return [];
```

Comment:
for whom is the empty array? :) 
returning empty array shouldn't have any effect besides returning the function.